tests/legacy-e2e/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/ShareWorkspaceMemberTest.java

```diff
@@ -43,16 +41,16 @@
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
-@Test(groups = {TestGroup.MULTIUSER, TestGroup.OPENSHIFT, TestGroup.K8S, TestGroup.UNDER_REPAIR})
+@Test(groups = {UNDER_REPAIR})
```

Comment:
You can just add `TestGroup.UNDER_REPAIR` item to the list not removing existed ones.